web/src/components/molecules/Common/LinkAssetModal/LinkAssetModal.tsx

```diff
@@ -49,6 +53,7 @@ const LinkAssetModal: React.FC<Props> = ({
   handleUpload,
 }) => {
   const t = useT();
+  const [hovered, setHovered] = useState<string>();
```

Comment:
Maybe `hoveredId` instead of `hovered` (same for set), hovered alone does not describe this variable's contents

Reply:
True. Changed to `hoveredAssetId` to be as clear as possible